resources/ajax_processing/insertIssue.php

```diff
@@ -83,12 +83,12 @@
 		unset($newIssueContact);
 	}
 
-	$organizationContactsArray = $sourceResource->organizationContactsArray($sourceOrganizationID);
+	$organizationContactsArray = $sourceResource->getUnarchivedContacts();
```

Comment:
getUnarchivedContacts is now a ResourceAcquisition method.
Use:
`$organizationContactsArray = $sourceResourceAcquisition->getUnarchivedContacts();`